src/admin-views/widgets/components/radio.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Admin View: Widget Radio Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/radio.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var string $label        Label for the radio group.
+ * @var string $value        Value for the radio group.
+ * @var string $button_value Value for the individual button.
+ * @var string $name         Name attribute for the radio.
+ * @var string $id           ID attribute for the radio.
+ * @var string $dependency   The dependency attributes for the control wrapper.
+ */
+
+?>
+<div
+	class="tribe-widget-form-control tribe-widget-form-control--radio"
+	<?php
+		// Not escaped - contains html ( data-attr="value")
+		echo $dependency;
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 675e92170d57fb670e48f091cb946027</sub>